src/pages/ServiceGraph/ServiceGraphPage.tsx

```diff
@@ -66,7 +68,9 @@ export default class ServiceGraphPage extends React.Component<RouteComponentProp
   }
 
   handleGraphClick = (data: any) => {
-    this.setState({ summaryData: data });
+    if (data !== undefined) {
```

Comment:
`undefined` is falsey so you can just use `if(data)`

https://www.sitepoint.com/javascript-truthy-falsy/